main/exercise/question.class.php

```diff
@@ -1957,29 +2152,29 @@ public static function deleteAllQuestionOptions($question_id, $course_id)
     /**
      * @param int   $id
      * @param array $params
-     * @param int   $course_id
+     * @param int   $courseId
      *
      * @return bool|int
      */
-    public static function updateQuestionOption($id, $params, $course_id)
+    public static function updateQuestionOption($id, $params, $courseId)
     {
         $table = Database::get_course_table(TABLE_QUIZ_QUESTION_OPTION);
         $result = Database::update(
             $table,
             $params,
-            ['c_id = ? AND id = ?' => [$course_id, $id]]
+            ['c_id = ? AND id = ?' => [$courseId, $id]]
         );
 
         return $result;
     }
 
     /**
      * @param int $question_id
-     * @param int $course_id
+     * @param int $courseId
      *
      * @return array
      */
-    public static function readQuestionOption($question_id, $course_id)
+    public static function readQuestionOption($question_id, $courseId)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $question_id is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)